src/views/RecIM/components/Helpers/index.js

```diff
@@ -1,35 +1,28 @@
-import { DateTime } from 'luxon';
+import { isValid, parseISO, format } from 'date-fns';
 
 const standardDate = (date, includeTime) => {
-  if (!DateTime.isDateTime(date)) date = DateTime.fromISO(date);
-  let formattedDate = date.monthShort + ' ' + date.day;
-  if (includeTime) {
-    formattedDate += ' ' + date.toLocaleString(DateTime.TIME_SIMPLE);
-  }
+  if (!isValid(date)) date = parseISO(date);
+  let formattedDate = includeTime ? format(date, 'MMM d h:mmaaa') : format(date, 'MMM d');
   return formattedDate;
 };
 
 const formatDateTimeRange = (startDateTime, endDateTime) => {
-  if (!DateTime.isDateTime(startDateTime)) startDateTime = DateTime.fromISO(startDateTime);
-  if (!DateTime.isDateTime(endDateTime)) endDateTime = DateTime.fromISO(endDateTime);
+  if (!isValid(startDateTime)) startDateTime = parseISO(startDateTime);
+  if (!isValid(endDateTime)) endDateTime = parseISO(endDateTime);
 
   // Jan 1 3:00pm - 6:00pm
-  let isOneDay =
-    startDateTime.toLocaleString(DateTime.DATE_FULL) ===
-    endDateTime.toLocaleString(DateTime.DATE_FULL);
+  let isOneDay = format(startDateTime, 'MM dd yyyy') === format(endDateTime, 'MM dd yyyy');
   if (isOneDay) {
-    let endTime = startDateTime.toLocaleString(DateTime.TIME_SIMPLE).replace(':00', '');
+    let endTime = format(endDateTime, 'h:mmaaa').replace(':00', '');
     return `${standardDate(startDateTime, true).replace(':00', '')} - ${endTime}`;
   }
   // Jan 1 3:00pm - Jan 31 3:00pm
-  let isOneYear = startDateTime.year === endDateTime.year;
+  let isOneYear = format(startDateTime, 'yyyy') === format(endDateTime, 'yyyy');
```

Comment:
Might be clearer and easier to compare the years as numbers instead of first converting them to strings:
```suggestion
  let isOneYear = startDateTime.getFullYear() === endDateTime.getFullYear();
```